### ISSUE TITLE
popover not positioned at element 

### ISSUE THREAD

**Issue by sysmat** [Kapsonfire-DE/svelte-ktippy#2]
popover not positioned at element  but tooltip is

- dependencies
```
{
"svelte-ktippy": "1.1.0"
 "bootstrap": "^4.6.0",
 "jquery": "^3.6.0",
 "popper.js": "^1.16.1",
 "svelte": "^3.37.0"
}
```  
- component:

```html
<script lang="ts">
  import { useFocus } from "svelte-navigator";
  import { tooltip, popover } from "svelte-ktippy";
  import Tooltip from "./Tooltip.svelte";

  const registerFocus = useFocus();

  const popoverKity = {
    component: Tooltip,
    arrow: true,
    props: { title: "my popover title", content: "My popover content" },
  };
</script>

<input use:registerFocus type="text" name="xyxyxy" />
<!-- TODO: implement-->
<div>Not implemented yet</div>
<br />
<div id="pop" use:popover={popoverKity}>popover</div>
<br />
<div
  use:tooltip={{
    component: Tooltip,
    arrow: true,
    props: { title: "my tooltip title", content: "My tooltip content" },
  }}
>
  tooltip
</div>
```
- tooltip:

```html
<script lang="ts">
  export let title: string;
  export let content: string;
</script>

<div class="card bg-dark text-white" style="width: 18rem;">
  <div class="card-body">
    <h5 class="card-title">{title}</h5>
    <p class="card-text">
      {content}
    </p>
  </div>
</div>
```

- screenshots

![kitypopover](https://user-images.githubusercontent.com/69830/116545680-00dcff00-a8f1-11eb-8237-5c36437d58ce.png)
![kityPopup](https://user-images.githubusercontent.com/69830/116545685-01759580-a8f1-11eb-9786-a5e824e5f28a.png)

**Comment by Kapsonfire** [comment]
Hello,

i set for popover placement property "top-start" per default, so it starts on the left of the element which is working fine.
the problem is you are using a div with a computed with of 100%;
you can set placement on your own 
> const popoverKity = {
>    component: Tooltip,
>    arrow: true,
>    placement: "top",
>    props: { title: "my popover title", content: "My popover content" },
>  };

also im currently changing this behaviour to "top" as default for 1.1.1

**Comment by sysmat** [comment]
Is this 'popoverKity'  a Interface or plain object?

**Comment by Kapsonfire** [comment]
fixed with https://github.com/Kapsonfire-DE/svelte-ktippy/commit/7932cddba66ca1da7509d046c318ff3ab5a6698a

**Comment by Kapsonfire** [comment]
released 1.1.1

**Comment by Kapsonfire** [comment]
> Is this 'popoverKity' a Interface or plain object?

it was copied from your source. its plain object. do you need an interface exported?

**Comment by sysmat** [comment]
It would be nice to know all possibilities

**Comment by Kapsonfire** [comment]
https://atomiks.github.io/tippyjs/v6/all-props/
in general i try to support all props given by tippyjs
But I agree: it should be documentated

**Comment by Kapsonfire** [comment]
Released Version 1.1.2 now exports a typescript interface KTippyProps @sysmat
